tests/fixtures/inversify/managedController.ts

```diff
@@ -5,15 +5,15 @@ import { TestModel } from '../testModel';
 import { ManagedService } from './managedService';
 
 @injectable()
-@Route('ManagedTest')
+@Route('{routeParam}/ManagedTest')
```

Comment:
why in the inversify controller?

Reply:
I just wanted a simple controller to test with. Could be expanded to other tests.